ghost/core/package.json

```diff
@@ -51,7 +51,7 @@
     "lint:code": "yarn lint:server && yarn lint:shared && yarn lint:frontend",
     "lint": "yarn lint:server && yarn lint:shared && yarn lint:frontend && yarn lint:test",
     "build:css": "postcss core/frontend/public/ghost.css --no-map --use cssnano -o core/frontend/public/ghost.min.css",
-    "prepack": "yarn workspace ghost-admin run build:prod && yarn build:css && npx --yes daniellockyer/monobundle"
```

Comment:
ghost-admin is built by the top level build command now & monobundle is installed locally for use on planes and linking locally for development

Reply:
Maybe we consider moving `yarn build:css` to the `build` script? So that prepack is just `monobundle`?

Reply:
> Maybe we consider moving yarn build:css to the build script? So that prepack is just monobundle?

Agree